README.md

```diff
@@ -67,6 +67,11 @@ CarbonData is built using Apache Maven, to [build CarbonData](https://github.com
 * [Carbon as Spark's Datasource](https://github.com/apache/carbondata/blob/master/docs/carbon-as-spark-datasource-guide.md) 
 * [FAQs](https://github.com/apache/carbondata/blob/master/docs/faq.md) 
 
+##  Integrations
+* [Hive](https://github.com/apache/carbondata/blob/master/docs/hive-guide.md)
+* [Presto](https://github.com/apache/carbondata/tree/master/integration/presto)
+* [Flink]( )
```

Comment:
It should be add after there are related flink documentation

Reply:
ok ,I get it